### PR TITLE
Save geom only if updated

### PR DESCRIPTION
**Pull request by Anaethelion**
Add new logo
Fix minors bugs
Add logs output on fail for travis
Fix output for imports hml/txt for command debug purposes

### CODE REVIEW

File: geotrek/common/parsers.py [hunk 1]
```diff
@@ -292,7 +292,8 @@ def report(self):
             'nb_unmodified': self.nb_unmodified,
             'warnings': self.warnings,
         }
-        return render_to_string('common/parser_report.html', context)
+        print(output_format)
```
Comment (leplatrem) [inline geotrek/common/parsers.py]: I recommend you to use logging :)

File: geotrek/common/parsers.py [hunk 2]
```diff
@@ -292,7 +292,8 @@ def report(self):
...
+        return render_to_string('common/parser_report.{output_format}'.format(output_format=output_format), context)
```
Comment (leplatrem) [inline geotrek/common/parsers.py]: In this case, using ``%s`` is acceptable

File: geotrek/common/parsers.py [hunk 3]
```diff
@@ -282,7 +282,7 @@ def parse_row(self, row):
         if self.progress_cb:
             self.progress_cb(float(self.line) / self.nb, self.line, self.eid_val)
 
-    def report(self):
+    def report(self, output_format='txt'):
```
Comment (leplatrem) [inline geotrek/common/parsers.py]: The txt format is not tested (see decreasing coverage)

File: geotrek/trekking/models.py [hunk 4]
```diff
@@ -269,6 +269,20 @@ def distance(self, to_cls):
     def is_public(self):
         return self.any_published or (self.parent and self.parent.any_published)
 
+    def save(self, *args, **kwargs):
+        if self.pk is not None and kwargs.get('update_fields', None) is None:
+            field_names = set()
+            for field in self._meta.concrete_fields:
+                if not field.primary_key and not hasattr(field, 'through'):
+                    field_names.add(field.attname)
```
Comment (leplatrem) [inline geotrek/trekking/models.py]: The update fields logics implemented here is not tested

File: geotrek/common/parsers.py (hunk 1 above)
Comment (Anaethelion) [inline geotrek/common/parsers.py]: This, is a mistake, good catch !

File: geotrek/common/parsers.py (hunk 3 above)
Comment (Anaethelion) [inline geotrek/common/parsers.py]: Will do.

File: geotrek/trekking/models.py (hunk 4 above)
Comment (Anaethelion) [inline geotrek/trekking/models.py]: It is the same logic as django's but i'll see what i can do.

File: geotrek/trekking/tests/test_views.py [hunk 5]
```diff
@@ -1064,3 +1064,29 @@ def test_can_publish(self):
         self.assertContains(response, 'Published')
         response = self.client.get('/trek/edit/%u/' % self.trek.pk)
         self.assertContains(response, 'Published')
+
+
+class TrekUpdateGeomTest(TestCase):
```
Comment (leplatrem) [inline geotrek/trekking/tests/test_views.py]: This is a detail, but this test belongs more to test_models :)

**Pull request merged**